dotCMS/src/main/java/com/dotmarketing/util/ConfigUtils.java

```diff
@@ -117,4 +119,32 @@ public static String getServerId(){
 	}
 
 
+    /**
+     * This method returns the absolute root path for assets
+     *
+     * @return the root folder of where assets are stored
+     */
+    public static String getAbsoluteAssetsRootPath() {
+        String realPath = Config.getStringProperty("ASSET_REAL_PATH", null);
+        if (UtilMethods.isSet(realPath) && !realPath.endsWith(java.io.File.separator)) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'java.io.File.separator' due to existing import 'java.io.File'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4756315)